modules/exploits/linux/local/sudo_chroot_cve_2025_32463.rb

```diff
@@ -0,0 +1,146 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = NormalRanking
+
+  include Msf::Post::File
+  include Msf::Post::Linux::Priv
+  include Msf::Post::Linux::System
+  include Msf::Post::Linux::Compile
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Sudo Chroot 1.9.17 Privilege Escalation',
+        'Description' => %q{
+          This exploit module illustrates how a vulnerability could be exploited
+          in an linux command for priv esc.
+        },
+        'License' => MSF_LICENSE,
+
+        'Author' => [
+          'msutovsky-r7', # module dev
+          'Stratascale', # poc dev
+          'Rich Mirch' # security research
+        ],
+        'Platform' => [ 'linux' ],
+
+        'Arch' => [ ARCH_CMD ],
+
+        # chmod has some issues for meterpreter, forcing shell
+        'SessionTypes' => [ 'shell' ],
+
+        'Targets' => [[ 'Auto', {} ]],
+
+        'Privileged' => true,
+
+        'References' => [
+          [ 'EDB', '52352' ],
+          [ 'URL', 'https://www.helpnetsecurity.com/2025/07/01/sudo-local-privilege-escalation-vulnerabilities-fixed-cve-2025-32462-cve-2025-32463/'],
+          [ 'CVE', '2025-32463']
+        ],
+        'DisclosureDate' => '2025-06-30',
+
+        'DefaultTarget' => 0,
+
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [ARTIFACTS_ON_DISK, IOC_IN_LOGS]
+        }
+      )
+    )
+
+    # force exploit is used to bypass the check command results
+    register_advanced_options [
+      OptString.new('WritableDir', [ true, 'A directory where we can write files', '/tmp' ]),
+
+    ]
+  end
+
+  # borrowed from exploits/linux/local/sudo_baron_samedit.rb
+  def get_versions
+    versions = {}
+    output = cmd_exec('sudo --version')
+    if output
+      version = output.split("\n").first.split(' ').last
+      versions[:sudo] = version if version =~ /^\d/
+    end
+    versions
+  end
```

Comment:
This could likely go into a mixin, as there are already two instances of this pattern in metasploit (see `git grep 'sudo --version' | wc -l`)